### ISSUE TITLE
[Feature] [doc] Update table schema for task-structure and metadata

### ISSUE THREAD

**Issue by zhongjiajie** [apache/dolphinscheduler#8628]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

I find out [task-structure](https://dolphinscheduler.apache.org/en-us/docs/latest/user_doc/architecture/task-structure.html) and [metadata](https://dolphinscheduler.apache.org/en-us/docs/latest/user_doc/architecture/metadata.html) table schema is out of dated. I would confuse users when they read them. I think we should update them in the next version release.

BTW, I think we should find a good way to auto-update the table schema to just do not show them in our doc. When some PR upgrade the schema by without update the doc, will make our doc out-of-date

### What you expected to happen

ATT

### How to reproduce

ATT

### Version

2.0.4

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by QuakeWang** [comment]
Please assign this to me, I will do it.

**Comment by zhongjiajie** [comment]
@QuakeWang  I assinged to you, but maybe we should find some ways to auto -generate table schema base on our SQL file to make our docs update from code. WDYT

**Comment by QuakeWang** [comment]
> @QuakeWang I assinged to you, but maybe we should find some ways to auto -generate table schema base on our SQL file to make our docs update from code. WDYT

Good idea! I will be looking for tools on how to automate the generation.

**Comment by zhongjiajie** [comment]
Thanks and I looking forward about it

**Comment by zhongjiajie** [comment]
remove stale
